src/vm-memory/src/mmap.rs

```diff
@@ -347,7 +404,7 @@ impl GuestMemoryMmap {
     /// Insert a region into the `GuestMemoryMmap` object and return a new `GuestMemoryMmap`.
     ///
     /// # Arguments
-    /// * `region`: the memory region to insert into the guest memory object.
+    /// * `region` - the memory region to insert into the guest memory object.
     pub fn insert_region(
```

Comment:
AFAIK we don't use this function. But we should handle the case where the new region doesn't have dirty_tracking enabled while the others have. Or the opposite.